R/utils.R

```diff
@@ -89,6 +89,15 @@ dir_create <- function(..., warn = TRUE, recurse = TRUE) {
   path
 }
 
+as_binary <- function(x) {
```

Comment:
suggestion: `as_bool()`

Reply:
Well, it does convert it to either `0` or `1`, rather than `TRUE` or `FALSE`?

Reply:
ha, yeah, in c it would be a bool, but it's fine the way it is.